jsonmatch/matcher.py

```diff
@@ -137,7 +143,9 @@ def _find_breaks(self, exp_d, test_d, key_trail, is_ordered, breaks):
             log.info("Someone is missing keys.")
             return breaks
 
-        for key, val in exp_d.items():
+        # PY3: `for key, val in exp_d.items()`
+        for key, val in iteritems(exp_d):
```

Comment:
if you `from builtins import dict` you can directly do `exp_d.items()`.  Though it is kind of inefficient in py2... 😓

Reply:
I'll leave this as it is, unless others disagree. While hotlanta has some large dicts but I don't think it matters much either way (just a gut check, no benchmark).